modin/engines/ray/generic/io.py

```diff
@@ -168,7 +165,7 @@ def read_parquet(cls, path, engine, columns, **kwargs):
             if directory:
                 # Path of the sample file that we will read to get the remaining
                 # columns.
-                pd = ParquetDataset(file_path)
```

Comment:
Do we need to look at all the files? We should be able to read the first file that we encounter for the remaining column names